### DISCUSSION TITLE
Github Pages from New Repo

### DISCUSSION THREAD

**Discussion by maisonml** [community/community#21788] [Other Feature Feedback, Questions, & Ideas]
<p>I had built a Github pages website from a public repo but then I wanted to make the repo private which I couldn’t because it was a forked repo.  So I upgraded to Github Pro, cloned this public repo to a private repo and renamed the private repo to the name of my public repo. But now the Github page is not getting built and am getting a 404 error. I waited for 20 minutes see if things needed time, but I still have this issue.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/maisonml">@maisonml</a>,</p>
<p>Thanks for being here! You will need to push a fresh commit before your site will build and serve again.</p>
<p>It can take up to 10 minutes to finish building your site, and another 10 to propagate across our CDN, so you may have to wait after enabling GitHub Pages or pushing a new commit for your site to show the latest changes.</p>
